src/components/Dropdown/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Size } from "@utilities/enum";
+import { ReactElement } from "react";
+import DropdownMenuItem, { MenuItem } from "../DropdownMenuItem/DropdownMenuItem";
+import css from "./DropdownMenu.module.css";
+
+export interface DropdownMenuProps {
+    items: MenuItem[][];
+    onChange: (id: string) => void;
+    size?: Size.Small | Size.Large;
+    activeItemId?: string;
+}
+
+export default function DropdownMenu({
+    items,
+    onChange,
+    size = Size.Small,
+    activeItemId = "",
+}: DropdownMenuProps): ReactElement<DropdownMenuProps> {
+    return (
+        <ul data-test-id="dropdown-menu" className={`${css.menu} ${size === Size.Large ? css.large : ""}`}>
```

Comment:
same as above

Reply:
Since there are only two different classes here I don't think the readablility would be improved by joining it together in an array. Therefore I'd leave it as it is.